### PR TITLE
Data Federation - Mongo DB

### PR DESCRIPTION
**Pull request by anna-liepina**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,23 +7,25 @@
     "node": ">=16"
   },
   "dependencies": {
+    "@types/mongodb": "^4.0.7",
```
Comment (eugene-matvejev) [inline package.json:10]: I don't think we need them here :)

File: src/parse:mongo.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env node
+
+require('dotenv');
+process.env.NODE_ENV = process.env.NODE_ENV || 'production';
```
Comment (eugene-matvejev) [inline src/parse:mongo.ts:4]: according last PR this line can go as well :)

File: package.json (hunk 1 above)
Comment (anna-liepina) [reply package.json]: fixed

**Review by eugene-matvejev — APPROVED** [review]
LGTM :+1: I tested works well!

**Pull request merged**